www/artists/dhall/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="stylesheet.css">
+
+</head>
+<body>
+  <ul>
+    <li class="links">  <a href= "index.html"> Home</a> </li>
+    <li class="links"> <a href= "photographs.html">photographs</a> </li>
+    <li class="links"> <a href= "https://saskartists.ca/">Sask artists</a> </li>
+  </ul>
+  <h1 class="Name">Don Hall</h1>
+  <p>Don hall has been a professional photographer since the 70's. He got interessted in photography from taking a drafting class in shop in high school. At that point he later  He later became interessted in how phtography as a medium speaks. His mother always carried a camera aroud and was an amateur photographer. Whenever someone asks him whats his favorite thing to take photos of he always answers Saskachewan, the people, the places, the environnement. He goes on road trips to take photos and he says the experience of going out and meeting new people is as important as the photographs he takes. His works have been featured in everywhere from the Mackenzie Art Gallery to the Smithsonian Mueseum and at one point or another his works have been featured in every mueseum in western canada. He loves photography and has loved photography and pursued it as a career for 49 years. He was Manager of the Photography Department at the University of Regina until 2014. </p>
+  <div style="display: flex; justify-content: center;">
```

Comment:
...What is the purpose of this inline CSS? Shouldn't stuff like this be kept in style.css?